modules/fedimint-wallet-server/src/lib.rs

```diff
@@ -760,6 +753,11 @@ pub struct Wallet {
     /// The result of last successful get_block_count
     block_count_local: std::sync::Mutex<Option<u32>>,
     our_peer_id: PeerId,
+    /// Block count updated periodically by a background task
+    /// Magic value `0` is used to denote it was not initialized yet.
+    block_count_bg: Arc<AtomicU32>,
+    /// Fee rate updated periodically by a background task
+    fee_rate_bg: Arc<RwLock<Option<Feerate>>>,
```

Comment:
You know my unreasonable fondness for `watch` channels @dpc :smile:, this could be:

```rust
fee_rate_bg: tokio::sync::watch::Receiver<Feerate>
```